spec.bs

```diff
@@ -2862,23 +2862,51 @@ partial interface Navigator {
   Promise<AdAuctionData> getInterestGroupAdAuctionData(AdAuctionDataConfig config);
 };
 
+dictionary AdAuctionData {
+  required Uint8Array request;
+  required USVString requestId;
+};
+</xmp>
+
+<xmp class="idl">
 dictionary AdAuctionDataConfig {
   required USVString seller;
   required USVString coordinatorOrigin;
   unsigned long requestSize;
   record<USVString, AdAuctionDataBuyerConfig> perBuyerConfig;
 };
+</xmp>
+
+<dl class=domintro>
+  <dt>{{AdAuctionDataConfig/seller}}
+  <dd>The seller that will be used as the {{AuctionAdConfig/seller}} in the
+    following {{AuctionAdConfig}} passed to {{Window/navigator}}.{{Navigator/runAdAuction()}}.
+  <dt>{{AdAuctionDataConfig/coordinatorOrigin}}
+  <dd>The origin of the coordinator hosting public encryption keys for the server
+    running the ad auction. The [=origin/scheme=] must be "`https`". An implementation
+    may select which coordinators are acceptable.
+  <dt>{{AdAuctionDataConfig/requestSize}}
+  <dd>The desired size for the returned response. If any buyers are specified in

```

Comment:
Let's be consistent about request/response here? I sometimes felt lost since the thing is called both request and response, although it is both in different context.
```suggestion
  <dd>The desired size for the returned {{AdAuctionData/request}}. If any buyers are specified in
```